### PR TITLE
Added get_era5

### PR DESCRIPTION
**Pull request by donal-mur**
Add file get_era5_data.py that uses the cdsapi  to get data from the Copernicus data centre. Changes were made to the new syntax for target file and data sets

### CODE REVIEW

File: src/scripts/get_era5_data.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python3
+"""
+A script that retrieves era-interim data
+from CDS data server. Note that a file
+named .ecmwfapirc, containg a key, should
```
Comment (BengtRydberg) [inline src/scripts/get_era5_data.py:5]: should be .cdsapirc, https://cds.climate.copernicus.eu/api-how-to

File: src/scripts/get_era5_data.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+be available in the user home directory.
+Updated 2020-03 för CDS server. Only
+pressure level part updated
+"""
+import os
+from sys import argv
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+import cdsapi
```
Comment (BengtRydberg) [inline src/scripts/get_era5_data.py:14]: cdsapi can be installed with pip, we must add this to requirements

File: src/scripts/get_era5_data.py [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            )
+            target = os.path.join(target_dir, target_file)
+            print(target)
+
+            # check if desired file already exists
+            # retrieve data if file not exists
+            if not os.path.exists(target):
+                pass
+                server = cdsapi.Client()
+                server.retrieve(service,cmd,target)
```
Comment (BengtRydberg) [inline src/scripts/get_era5_data.py:164]: should be something like:
dataset = "reanalysis-era5-pressure-levels"
server.retrieve(dataset, cmd, target) 
according to https://cds.climate.copernicus.eu/api-how-to

have verified that files are retrieved with these changes

**Comment by joakimmoller** [comment]
For operational use, @donal-mur  have place the key in ~opdinop/.cdsapirc  (according to the how to)

**Comment by local-minimum** [comment]
The reason the tests fail are the following style errors:
```
./src/scripts/get_era5_data.py:26:8: E225 missing whitespace around operator

./src/scripts/get_era5_data.py:164:33: F821 undefined name 'service'

./src/scripts/get_era5_data.py:164:40: E231 missing whitespace after ','

./src/scripts/get_era5_data.py:164:44: E231 missing whitespace after ','

```

**Comment by joakimmoller** [comment]
this PR was superseded by #23

**Pull request closed without merging**